app.js

```diff
@@ -0,0 +1,10 @@
+import config from "./config/config";
+import { User } from "./models/User";
```

Comment:
Both models `User` and `Product` should be imported with one `import` statement. Please recheck 7th point of task

Reply:
Done.